packages/client/tests/functional/binary-engine/reuse-binary-engine/prisma/_schema.ts

```diff
@@ -4,7 +4,6 @@ import testMatrix from '../_matrix'
 export default testMatrix.setupSchema(({ provider }) => {
   return /* Prisma */ `
     generator client {
-      engineType = "binary"
```

Comment:
Small cleanup, this should never be hardcoded (and was ignored because of our reliance on the engine being selected via the env var).